bench/wtperf/wtperf.c

```diff
@@ -979,28 +1144,36 @@ run_mix_schedule(WTPERF *wtperf, WORKLOAD *workp)
 		return (0);
 	}
 
-	/* Confirm reads, inserts and updates cannot all be zero. */
-	if (workp->insert == 0 && workp->read == 0 && workp->update == 0) {
+	/* Confirm reads, inserts, modifies and updates cannot all be zero. */
+	if (workp->insert == 0 && workp->read == 0 && workp->update == 0 &&
+	    workp->modify == 0) {
```

Comment:
Alphabetical (in pretty much all the places here and below).

Reply:
corrected.